engine/src/main/java/org/apache/hop/core/IProvidesDatabaseConnectionInformation.java

```diff
@@ -23,6 +23,7 @@
  * An interface for pipeline transforms that connect to a database table. For example a table output
  * transform or a bulk loader.
  */
+@Deprecated(since = "2.9", forRemoval = true)
```

Comment:
this should be 2.10 as it was not deprecated in 2.9

Reply:
My bad I forget to change it. Thanks for pointing me there